### PR TITLE
Towards re-enabling unidoc

### PR DESCRIPTION
**Pull request by raboof**
`sbt -Dakka.genjavadoc.enabled=true ++2.12.3 unidoc` seems to produce scaladoc and javadoc now, not sure what to look for beyond that.

(remaining jenkins error is formatting)

### CODE REVIEW

File: project/Release.scala [hunk 1]
```diff
@@ -1,45 +1,44 @@
...
+    IO.createDirectory(release)
+    IO.copyDirectory(repo, release / "releases")
+    IO.copyDirectory(api, release / "api" / "akka" / releaseVersion)
+    IO.copyDirectory(japi, release / "japi" / "akka" / releaseVersion)
+    IO.copyDirectory(docs, release / "docs" / "akka" / releaseVersion)
+
+    state3
+  }
+
+}
```
Comment (ktoso) [inline project/Release.scala]: exciting :-)

**Comment by 2m** [comment]
I have tried to build docs with this PR and it seems that javadoc is not generated. Directory `target/javaunidoc` is empty.

**Comment by raboof** [comment]
Already applied to `jdk9-build`

**Comment by ktoso** [comment]
Good:)

**Pull request closed without merging**